### PR TITLE
Maestro colour and logo

### PR DESCRIPTION
**Pull request by Beat-tech**
I have added Maestro credit car colour theme and logo. Thank you so much for taking beginners into consideration!!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,5 +27,15 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Beat-tech/credit-card-form.git"
+  },
+  "author": "Beat-tech",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Beat-tech/credit-card-form/issues"
+  },
+  "homepage": "https://github.com/Beat-tech/credit-card-form#readme"
```
Comment (riandy-dimas) [inline package.json]: I don't think this change is necessary, can you help to remove this? @Beat-tech

File: src/utils.js [hunk 2]
```diff
@@ -34,6 +34,8 @@ function getCardColorByType(cardType = "other") {
     case cards.amex:
       color = "rgb(0, 112, 209)";
       break;
+    case cards.maestro:
+      color = "rgb(196, 81, 225)";
```
Comment (riandy-dimas) [inline src/utils.js:38]: Please add `break` statement here ya @Beat-tech

**Review by riandy-dimas — CHANGES_REQUESTED** [review]
Great job! But it seems some changes must be done to make sure the application runs well.

File: package.json (hunk 1 above)
Comment (Beat-tech) [reply package.json]: Ok, on it!

**Comment by riandy-dimas** [comment]
Also make sure that the credit card will change into the Maestro card theme you have been defined when inserting a dummy card number input. (e.g for Maestro: 6759649826438453) @Beat-tech thanks in advance!

File: src/utils.js (hunk 2 above)
Comment (Beat-tech) [reply src/utils.js:38]: On it!

File: package.json (hunk 1 above)
Comment (riandy-dimas) [reply package.json]: ehmm @Bear-tech I mean just to remove the `,` in the line `30` and the rest of the code from line `31` until `40`, not to remove entire `package.json` file 😅 removing `package.json` file will breaking the app *FYI

**Comment by Beat-tech** [comment]
Do you need me to try a number in the following function? Sorry didn't understand the last comment quite well.

`function App() {
  const [name, setName] = useState("Riandy Dimas");
  const [number, setNumber] = useState("371209512822287");
  const [date, setDate] = useState({ month: "02", year: "22" });
  const [cvv, setCvv] = useState("022");
  const [flipped, setFlipped] = useState(false);
`

**Comment by riandy-dimas** [comment]
> Do you need me to try a number in the following function? Sorry didn't understand the last comment quite well.
> 
> `function App() { const [name, setName] = useState("Riandy Dimas"); const [number, setNumber] = useState("371209512822287"); const [date, setDate] = useState({ month: "02", year: "22" }); const [cvv, setCvv] = useState("022"); const [flipped, setFlipped] = useState(false); `

oh, can you run this app on your local machine? please read [this file](https://github.com/riandy-dimas/credit-card-form/blob/main/README.md) to do so. After successfully running your app in your local device, try to input Maestro card number in the respective field. Something like this will appear:
image

**Comment by Beat-tech** [comment]
I can't run it, not even following the instructions. I don't know why. Sorry and thank you for your patience!
I will try later.

File: package.json (hunk 1 above)
Comment (Beat-tech) [reply package.json]: Ok, gonna fork it again and start over. You are so kind, thank you really!

**Comment by riandy-dimas** [comment]
That's fine @Beat-tech everybody has their own starting point.  If you still interested in doing this task, you can see [this pull](https://github.com/riandy-dimas/credit-card-form/pull/6) request for your example. You only have to do these changes:
- create a new pull request
- update `src/card/styles.scss`
- update `src/utils.js`
- add a new logo in the `src/assets` folder
- and make sure not to modify the other files

The problem that might be happened (you can not run the app) is you delete the `package.json` file. Please let me know whether you still interested or not, okay? 😁

**Comment by Beat-tech** [comment]
> That's fine @Beat-tech everybody has their own starting point. If you still interested in doing this task, you can see [this pull](https://github.com/riandy-dimas/credit-card-form/pull/6) request for your example. You only have to do these changes:
> 
> * create a new pull request
> * update `src/card/styles.scss`
> * update `src/utils.js`
> * and add a new logo in the `src/assets` folder
> 
> The problem that might be happened (you can not run the app) is you delete the `package.json` file. Please let me know whether you still interested or not, okay?

Yes!! I am still interested if that's not too much for ask. :)

**Comment by riandy-dimas** [comment]
> > That's fine @Beat-tech everybody has their own starting point. If you still interested in doing this task, you can see [this pull](https://github.com/riandy-dimas/credit-card-form/pull/6) request for your example. You only have to do these changes:
> > 
> > * create a new pull request
> > * update `src/card/styles.scss`
> > * update `src/utils.js`
> > * and add a new logo in the `src/assets` folder
> > 
> > The problem that might be happened (you can not run the app) is you delete the `package.json` file. Please let me know whether you still interested or not, okay?
> 
> Yes!! I am still interested if that's not too much for ask. :)

Nice! You can restart again by making a new pull request if you want, or just continue in this PR, happy code!

**Comment by riandy-dimas** [comment]
It seems that this PR is inactive for too long, so I decided to close it

**Pull request closed without merging**